internal/client/client.go

```diff
@@ -100,30 +105,33 @@ func Run(addr, fingerprint, proxyAddr string) {
 
 	sshPriv, sysinfoError := keys.GetPrivateKey()
 	if sysinfoError != nil {
-		log.Fatal("Getting private key failed: ", sysinfoError)
+		log.Println("Getting private key failed: ", sysinfoError)
 	}
 
 	l := logger.NewLog("client")
 
-	var username string
-	userInfo, sysinfoError := user.Current()
-	if sysinfoError != nil {
-		l.Warning("Couldnt get username: %s", sysinfoError.Error())
-		username = "Unknown"
-	} else {
-		username = userInfo.Username
+	if username == "" {
```

Comment:
This is confusing, as it says "if I have a username specified then use the specified user name otherwise get the system username" which makes very little sense, and again hides the real world user that the RSSH client starts in, which reduces the featureset of RSSH

Reply:
Really start the user, once we are connected, we can still get a lot of commands, here is to support, compile an independent user name and password for each client

Reply:
I am not keen on having the clients have their own username and password to authenticate to the server. It adds no benefit over public key authentication and is in-fact worse.

Reply:
if you no need this feature ,just keep build without pass username and password,nothing will be changed